src/content/docs/apis/nerdgraph/examples/manage-accounts-nerdgraph.mdx

```diff
@@ -0,0 +1,149 @@
+
+## Before you start [#concepts]
+
+Before you use NerdGraph to manage your accounts, it will help if you understand:
+
+* [What New Relic accounts are and what they're used for](/docs/accounts/accounts-billing/account-structure/new-relic-account-structure)
+* [The basics of using NerdGraph](/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph)
+
+This tutorial will show you how to: 
+* View accounts 
+* Rename accounts
+* Add accounts
+
+## Tips on terms and fields [#field-definitions]
+
+Some tips about some of the fields used in these requests: 
+* The `managedAccount` and `managedAccounts` fields are simply terms for the accounts in an organization. They're synonymous with "accounts."
+* The `regionCode` field refers to [the data center used by an account](/docs/accounts/accounts-billing/account-setup/choose-your-data-center): `US` or `EU`. Note that an organization can only use a single data center: there can't be accounts that use diffrent data centers in a single organization. 
+
+Note that the [NerdGraph explorer](/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph/#explorer) has built-in docs that defines fields.
+
+## View accounts [#view-accounts]
+
+Here’s an example of returning the accounts in your organization: 
+
+```
+{
+  actor {
+    organization {
+      accountManagement {
+        managedAccounts {
+          name
+          id
+          regionCode
+        }
+      }
+    }
+  }
+}
+```
+
+## Create accounts [#create-accounts]
+
+Here’s an example of how to create an account. Before creating an account, make sure you understand [what an account is used for](/docs/accounts/accounts-billing/account-structure/new-relic-account-structure). 
+
+```
+mutation {
+  accountManagementCreateAccount(managedAccount: {name: "<var>NEW_ACCOUNT_NAME</var>"}) {
+    managedAccount {
+      id
+      name
+      regionCode
+    }
+  }
+```
+
+## Rename an account [#rename-accounts]
+
+Here’s an example of how to rename an account. 
+
+```
+mutation {
+  accountManagementUpdateAccount(managedAccount: {name: "<var>UPDATED_ACCOUNT_NAME</var>", id: <var>101010101</var>}) {
+    managedAccount {
+      id
+      name
+      regionCode
+    }
+  }
+}
+```
+
+## Errors [#errors]
+
+Here are some errors and what they mean: 
+
+    <table>
+      <thead>
+        <tr>
+          <th className="fcenter">
+            **Error message**
+          </th>
+
+          <th className="fcenter">
+            **Likely cause**
+          </th>
+        </tr>
+      </thead>
+
+      <tbody>
+        <tr>
+          <td>
+```"unauthorized", {
+      extensions: { nerdGraphExtensions: { errorClass: "ACCESS_DENIED" } }
+    }
+```
+</td>
+
+<td>
+This points to a problem with your organization structure (for example, an account that is outside the boundaries of an organization). If you get this message, talk to [support](/docs/new-relic-solutions/solve-common-issues/find-help-use-support-portal) or your account representative. 
```

Comment:
Consider changing "talk to" to "contact" because there may not be phone or zoom involved :-)